README.md

```diff
@@ -70,7 +70,7 @@ A Terraform Module to configure the Lacework Agentless Scanner.
 | <a name="input_bucket_enable_ubla"></a> [bucket_enable_ubla](#input_bucket_enable_ubla)                                                                            | Boolean for enabling Uniform Bucket Level Access on the created bucket. Default is `true`.        | `bool`                                                                                                                                                                                                                                                                         | `true`                                                                                                                                                                                                                                                 |    no    |
-| <a name="input_bucket_force_destroy"></a> [bucket_force_destroy](#input_bucket_force_destroy)                                                                      | Force destroy bucket (Required when bucket not empty)                                             | `bool`                                                                                                                                                                                                                                                                         | `true`                                                                                                                                                                                                                                                 |    no    |
+| <a name="input_bucket_force_destroy"></a> [bucket_force_destroy](#input_bucket_force_destroy)                                                                      | Force destroy bucket (if disabled, terraform will not be able do destroy non-empty bucket)        | `bool`                                                                                                                                                                                                                                                                         | `true`                                                                                                                                                                                                                                                 |    no    |
```

Comment:
nit: `if disabled, terraform will not be able do destroy non-empty bucket` -> `if disabled, terraform will not be able to destroy non-empty bucket`

do -> to